### ISSUE TITLE
Alpha banner

### ISSUE THREAD

**Issue by dsamojlenko** [cds-snc/c19-benefits-node#200]
The prominent-looking “ALPHA” text in the bar at the top of the page isn’t coded in a way that makes that obvious to screen readers. I’d recommend placing “ALPHA” in a <strong> element, adjusting CSS to prevent the text from becoming visually-bolded (the black border already distinguishes it enough) and adding a visually-hidden colon before the sentence. That way, screen readers will see it as “ALPHA: This site will change as we test ideas.”

**Comment by katedee** [comment]
Makes sense, I would add that the ALPHA text is _not_ bolded
